.github/actions/run-shared-unit-tests/requirements.txt

```diff
@@ -0,0 +1,7 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
It is probably good to move this file to the `tests` directory and do the respective changes in the `CONTRIBUTING.md` file.